site/sfguides/src/credit-card-fraud-detection-using-snowflake-ml/credit-card-fraud-detection-using-snowflake-ml.md

```diff
@@ -0,0 +1,176 @@
+author: Kala Govindarajan
+id: credit-card-fraud-detection-using-snowflake-ml
+summary: Build a Credit card fraud detection app that leverages Snowflake ML to train a binary classification model to detect if a transaction is fraudulent or not
+categories: Data-Science-&-Ml
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Snowflake ML Function, Data Science, Feature Store, Machine Learning
+
+# Credit Card Fraud Detection using Snowflake ML Functions
+<!-- ------------------------ -->
+
+## Overview
+Duration: 1
+
+### Overview
+In this Quickstart guide, we will train a model using Snowflake ML Functions for Credit card fraud detection. Fraud Detection involves a range of methods and techniques aimed at preventing fraudulent purchases, both online and in-store. The goal is to verify the legitimacy of the transaction and ensure that the true cardholder is involved. Depending on the detection techniques used one may address questions about the cardholder's identity and intentions either in real-time or after the transaction has occurred. Thus, credit card fraud detection serves as both a preventative measure against payment fraud and a method for investigating past transactions. We will be walking through a use case where the ML Capabilities of the Snowflake Data Cloud will be leveraged for investigating the past transactions.
+
+### What You'll Build
+A pipeline to implement Feature Engineering and storing the features in a Feature Store inside Snowflake Notebooks
+A pipeline to build a binary classification model to predict if a transaction is fraudulent or not inside Snowflake Notebooks
+A Streamlit application with geospatial analysis that will be used to carry ongoing inference by detecting if new transactions are fraudulent or not.
+
+### What You'll Learn
+How to build a Model using Snowflake ML Classification Function
+How to build a Streamlit app to detect and visualize the fraudulent transactions in Snowflake
+How to render a map and plot in the Streamlit app
+
+### What You'll Need
+- Snowflake account with a SYSADMIN role or a role that has CREATE SNOWFLAKE.ML.CLASSIFICATION privilege on the schema the model will be created
+- A GitHub Account
+- Intermediate knowledge of Snowsight, the UI interface of Snowflake
+
+# Setup
```

Comment:
broken syntax. Please render this file locally to ensure proper step structure.

Reply:
Thanks Daniel,  Just fixed that.. Please review when you get a chance.